### PR TITLE
P0892R2 explicit(bool)

### LINKED ISSUE #2128: [2018-06 CWG Motion 16] P0892R2 explicit(bool)

### PR DESCRIPTION
**Pull request by jensmaurer**
Fixes #2128.

### CODE REVIEW

File: source/special.tex [hunk 1]
```diff
@@ -103,7 +103,7 @@
 The \grammarterm{class-name} shall not be a \grammarterm{typedef-name}. In a constructor
 declaration, each \grammarterm{decl-specifier} in the optional
 \grammarterm{decl-specifier-seq} shall be \tcode{friend}, \tcode{inline},
-\tcode{explicit}, or \tcode{constexpr}.
+or \tcode{constexpr} or an \grammarterm{explicit-specifier}.
```
Comment (zygoloid) [inline source/special.tex]: The phrasing here is a bit weird, this parses as "shall be (friend), (inline), or (constexpr or an explicit-specifier)". How about:

> shall be `friend`, `inline`, `constexpr`, or an *explicit-specifier*.

File: source/compatibility.tex [hunk 2]
```diff
@@ -1892,6 +1892,26 @@
 };
 \end{codeblock}
 
+\rSec2[diff.cpp17.decl]{\ref{dcl.decl}: Declarators}
```
Comment (zygoloid) [inline source/compatibility.tex]: Declarators -> declarators

Comment (jensmaurer) [reply source/compatibility.tex]: Fixed.

File: source/special.tex (hunk 1 above)
Comment (jensmaurer) [reply source/special.tex]: Fixed.

**Pull request merged**